x-pack/legacy/plugins/apm/public/hooks/useFetcher.tsx

```diff
@@ -52,9 +52,7 @@ export function useFetcher(
   } = {}
 ) {
   const {
-    notifications: {
-      toasts: { addWarning }
-    }
+    notifications: { toasts }
   } = useKibanaCore();
```

Comment:
Not something you have to change here but personally I find nested destructurings harder to read, and prefer:
```js
const { notifications } = useKibanaCore();

// ...

notifications.toasts.addWarning({));
```

A surprising side-effect is that it's fewer lines